compose-builder/add-device-gpio.yml

```diff
@@ -31,7 +31,6 @@ services:
       - device-common.env
     environment:
       SERVICE_HOST: edgex-device-gpio
-      SERVICE_PORT: 59994
```

Comment:
why this env is not needed for older version?

Reply:
This was added when we were going to fix the port used to match the value assigned and document. Change of direction was to fix the docs  to use the actual port used instead.

Reply:
got it